### ISSUE TITLE
Ability for a moderator to pin/highlight/spotlight a participant as speaker in a call

### ISSUE THREAD

**Issue by small1** [nextcloud/spreed#5572]
**Is your feature request related to a problem? Please describe.**
When using several displays in a conference room one would like to have the ability to highlight/pin a specific user or a couple of users for all the participants in a video call.   A speaker/object (like a whiteboard or similar) that regardless of whom is speaking is always visible. 

**Describe the solution you'd like**
Moderator should have an option to pin/ghtliught a specific particpant for all users in a call. Much like the presentation mode does when someone has a presentation.

**Additional context**
This is a feature available in zoom and is called spotlight. A moderator can spotlight up to 9 particpants that is always shows regardless of what the users use as a layout.   This is handy if you have one or two speaker that has a presentation or an active interview that is going to be watched by the others.

**Comment by daniel4999** [comment]
Would love to see this feature as well

**Comment by pdmarf** [comment]
yes please! I hate having to tell people to how to select the 'speaker view' and would love to just turn it on/off for all viewers. 'spotlight' in zoom is a great feature that I miss in Talk.
